src/routes/(console)/project-[region]-[project]/functions/create-function/manual/+page.svelte

```diff
@@ -153,6 +160,15 @@
               };
           })
         : [];
+
+    $: maxSize =
+        isCloud && $currentPlan
+            ? $currentPlan?.deploymentSize === 0
+                ? 0
```

Comment:
Why/when would this be 0? Do we even need this if because it deployment size was 0, 0 * 1000000 is also 0.

Reply:
for Unlimited, it is 0 on the BE based on a given plan.

Reply:
Oh I didn't realize we had a plan with unlimited.

Anyways, in either case for the if, if deployment size was 0, the result would be 0 wouldn't it?

Reply:
So what other case is this if supposed to cover?

Reply:
If not `0`, we still need to `* 1000000` because the value from BE is not a full sized byte value, just simpler integer like `30`/`60`, etc..

Reply:
But if deployment size is 0, 0 * 1000000 is 0, so why do we need the if?

Reply:
aah that way. makes sense. will fix.

Reply:
yepp.